generators/angular/templates/src/main/webapp/app/config/language.constants.ts.ejs

```diff
@@ -21,5 +21,5 @@
     They are written in English to avoid character encoding issues (not a perfect solution)
 */
 export const LANGUAGES: string[] = [
-    // jhipster-needle-i18n-language-constant - JHipster will add/remove languages in this array
+  // jhipster-needle-i18n-language-constant - JHipster will add/remove languages in this array
```

Comment:
Are needles still considered compared to two-step transform? I don't get all the thing, but the later seems to be non-invasive. WDYT of creating a ticket to keep only one of the two approchaes

Reply:
They have different purposes.
Two-step transform is more complex.

In this case we have `jhipster languages` command that injects languages in existing application and requires needles to work.